lib/Unix/PID/Tiny.pm

```diff
@@ -242,9 +331,34 @@ Like Unix::PID but supplies only a few key functions.
 
 =head1 INTERFACE
 
-=head2 new()
+=head2 new(I<[$args_hr]>)
+
+See L<Unix::PID>'s new().  The following options can be provided in the
+optional HASH, I<$args_hr>, to enable certain extensions:
+
+=over
+
+=item B<keep_open>
+
+When a true value is provided, PID files created with pid_file() will remain
+open.
 
-See L<Unix::PID>'s new()
+=item check_proc_open_fds
+
+When a true value is provided, this option will cause is_pidfile_running() to
+traverse C</proc/$pid/fd> to ensure a current file descriptor is held by a
+given process for the specified PID file.
+
```

Comment:
> =item check_proc_open_fds
> …
> This option is only useful when B<keep_open> is also enabled.

`check_proc_open_fds` doesn’t seem to care about `keep_open`. I think that last line of POD needs removed no?

Reply:
It doesn't have to. Again, something other than Unix::PID::Tiny could have created the pid file, a use case I would like to make non-painful to support.

Reply:
So if `check_proc_open_fds` doesn’t use `keep_open` then why is it documented  that `This option is only useful when B<keep_open> is also enabled.`. 

It is useful by itself no matter the state of `keep_open`, right?

Reply:
I've updated the POD to reflect this possible use case.